constants.yml

```diff
@@ -200,6 +200,8 @@ encounterType:
     uuid: "5021b1a1-e7f6-44b4-ba02-da2f2bcf8718"
   REHAB_EVAL:
     uuid: "e91a4139-e0e7-447f-a5dd-c4f3b92d27c9"
+  NURSE_CONSULT:
+    uuid: "e920911f-dee9-4b18-9a58-50e685806ea0"
```

Comment:
extract the uuid for the Nurse Consult out into a variable so that it will be compiled and availalbe in code